maestro.py

```diff
@@ -162,7 +168,7 @@ def process_info_message(message):
 
 def on_message(ws, message):
     message_array = message.split("|")
-    if message_array[0] == MaestroMessageType.Info.value:
+    if message_array[0] == MaestroMessageType.Info.value | message_array[0] == MaestroMessageType.String.value:
```

Comment:
Not absolutely shure if it's the correct way to handle it. The stove sends back a message:
`42["rispondo",{"stringaRicevuta":"AA|171220202354","socketChiamata":"LvpI_guuJgmxzryC5qQr"}]`

Reply:
Dont think you can do anything with that. Here's the code that i found from the android app, it justs eats the stringrecvutta "AA" response.

```
function elaboraStringaRicevuta( stringaPassata )
{
    var datiRicevuti = stringaPassata.split("|");
    var end = new Date().getTime();
    var stringa = stringaPassata.substring(3);
    var dispositivoConnesso = $("#sub-lista-dispositivi .cambia-dispositivo[mac='" + dispositivoAssociato[1] + "'] .img-selected");
    var dispositivoConnesso2 = $(".popup-modifiche .dispositivi-associati .cambia-dispositivo[mac='" + dispositivoAssociato[1] + "'] .img-selected")
    
    if( datiRicevuti[0] == "PING" )
            {$( "#sub-info .recupero-stato-array .recupero-versione-pannello-direct .valore-recuperato" ).html( datiRicevuti[1] ); sendCommand( "PONG", "PINGPONG" );}
    else if( datiRicevuti[0] == "00" && stringa.length > 0 ) 
        {statoConnessione = 30; funzioniGenerali(stringa, "00"); nascondiDivAlert(); statoConnessioneOld = 20; $(".popup-modifiche .close-popup").trigger("click");}
    /*else if( datiRicevuti[0] == "00" ) // 
        {
            if( tipoConnessione == "direct" ) {statoConnessione = 100; console.log("00 corta:");}
            if( tipoConnessione == "remoto" ) {statoConnessione = 20; statoConnessioneTime = new Date().getTime();}
        }*/
    else if( datiRicevuti[0] == "01" && stringa.length >= 0 )
        {
         
            statoConnessione = 40; timeChiamate = new Date().getTime(); funzioniGenerali(stringa, "01"); 
            if( statoConnessioneOld == 20 )
                {
                    dispositivoConnesso.html('<img src="images/stove.png" width="100%" />');
                    dispositivoConnesso2.html('<img src="images/stove.png" width="100%" />');
                    if( sistemaOperativo == 0 ) {
                        navigator.notification.activityStop();
                    } 
                    statoConnessioneOld = 30;
                }
            //if( primaAperturaAPP == 0 ) {$("#sub-power").removeClass("hidden"); $("#sub-lista-dispositivi").addClass("hidden"); primaAperturaAPP = 1;}
        }
    /*else if( datiRicevuti[0] == "01" )  // 
        {
            if( tipoConnessione == "direct" ) {statoConnessione = 100; console.log("01 corta:");}
            if( tipoConnessione == "remoto" ) {statoConnessione = 30;}
        }*/
    else if( datiRicevuti[0] == "AA" )
        {}
    else if( stringa.length > 0 ) 
        {funzioniGenerali(stringa, datiRicevuti[0], datiRicevuti[1]); $("#xLoader").fadeOut("slow");}
    /*else if( stringa.length == 0 )
        {
            if( tipoConnessione == "direct" ) {statoConnessione = 100; console.log("Stringa Vuota:");}
            if( tipoConnessione == "remoto" ) {socket.emit('disconnect'); statoConnessioneTime = new Date().getTime() - 7000; statoConnessione = 20;}
        }*/
    
    console.log("Stato: " + statoConnessione + " - MS:", end - start, " - Stringa: " + stringa );
}

```